pkg/kubelet/server/server_test.go

```diff
@@ -575,6 +576,72 @@ func TestAuthzCoverage(t *testing.T) {
 	}
 }
 
+func TestInstallAuthNotRequiredHandlers(t *testing.T) {
+	fw := newServerTestWithDebug(false, nil)
+	defer fw.testHTTPServer.Close()
+
+	// No new handlers should be added to this list.
+	allowedAuthNotRequiredHandlers := sets.NewString(
+		"/healthz",
+		"/healthz/log",
+		"/healthz/ping",
+		"/healthz/syncloop",
+		"/metrics",
+		"/metrics/slis",
+		"/metrics/cadvisor",
+		"/metrics/probes",
+		"/metrics/resource",
+		"/pods/",
+		"/stats/",
+		"/stats/summary",
+	)
+
+	// These handlers are explicitly disabled.
+	debuggingDisabledHandlers := sets.NewString(
+		"/run/",
```

Comment:
I don't see checkpoint in either of the lists anymore. Is that expected?

Reply:
Yeah, we don't have coverage of every endpoint in the debugging disabled handlers. TBH I don't know why the debugging disabled handlers exist.

Reply:
Looks like it was added to address https://github.com/kubernetes/kubernetes/issues/34457